### PR TITLE
Update new-staff-guide.md

### PR DESCRIPTION
**Pull request by justinegeffen**
Fixes https://github.com/mattermost/mattermost-developer-documentation/issues/349

### CODE REVIEW

File: site/content/internal/onboarding/new-staff-guide.md [hunk 1]
```diff
@@ -24,20 +24,20 @@ weight: 10
...
-- You should be added to the private team `Private Core` and join the following channels
+- You should be added to the private team `Staff` and join the following channels
     - Alerts
     - Announcements
-    - Community/Customers
+    - Customer Success
+    - Customer Support
+    - Confidential Bugs
     - New CLA Entries
-    - Platform Meeting
```
Comment (hanzei) [inline site/content/internal/onboarding/new-staff-guide.md]: This should be R&D Meeting

File: site/content/internal/onboarding/new-staff-guide.md [hunk 2]
```diff
@@ -24,20 +24,20 @@ weight: 10
...
 ### Channels and Teams
-- You should be added to the private team `Private Core` and join the following channels
+- You should be added to the private team `Staff` and join the following channels
     - Alerts
     - Announcements
-    - Community/Customers
+    - Customer Success
+    - Customer Support
+    - Confidential Bugs
     - New CLA Entries
```
Comment (hanzei) [inline site/content/internal/onboarding/new-staff-guide.md]: I don't see the value of being a member of this channel.

File: site/content/internal/onboarding/new-staff-guide.md [hunk 3]
```diff
@@ -24,20 +24,20 @@ weight: 10
...
-    - Platform Meeting
     - Private Off-Topic (optional)
     - Social: * (optional)
-    - Standup
+    - Stand-up
     - Welcome
 - Private channels to join in `Contributors`
     - Security
-    - Developer: Private
-    - Confidential Bugs
```
Comment (hanzei) [inline site/content/internal/onboarding/new-staff-guide.md]: This is correct. There is a `Confidential Bugs` channel in `Contributors'.

File: site/content/internal/onboarding/new-staff-guide.md [hunk 4]
```diff
@@ -24,20 +24,20 @@ weight: 10
...
+    - Developers: Private
```
Comment (hanzei) [inline site/content/internal/onboarding/new-staff-guide.md]: You should also join `MVP Discussion`.

**Comment by hanzei** [comment]
@justinegeffen Thanks for cleaning up all of this old issues!

**Comment by justinegeffen** [comment]
> @justinegeffen Thanks for cleaning up all of this old issues!

No problem - glad to help. :)

File: site/content/internal/onboarding/new-staff-guide.md (hunk 4 above)
Comment (amyblais) [reply site/content/internal/onboarding/new-staff-guide.md]: I would propose not adding MVP Discussion. It's a private channel so people can't join it. I add new R&D folks when they are announced.

**Pull request merged**